egui/src/containers/collapsing_header.rs

```diff
@@ -188,6 +192,19 @@ impl CollapsingHeader {
         self.enabled = enabled;
         self
     }
+
+    /// Can the `CollapsingHeader` be selected by clicking it? Default: `false`.
+
```

Comment:
I think we should also have some example code, as it is not obvious how to check if the header was selected.

```suggestion
    /// Can the `CollapsingHeader` be selected by clicking it? Default: `false`.
    ///
    /// ``` rust
    /// # let ui = &mut egui::Ui::__test();
    /// let mut selected = false;
    /// let response = CollapsingHeader::new("Select and open me")
            .selectable(true)
            .selected(selected)
            .show(ui, |ui| ui.label("Content"));
    /// if response.header_response.clicked() {
    ///     selected = true;
    /// }
    /// ```
```